html/register_credentials.html

```diff
@@ -23,6 +34,16 @@
             <br>
             <input name="phoneNumber">
         </label><br>
-        <button>OK</button>
+        <label>
+            Datenschutz:
+             <br>
+            <input type="checkbox" name="dataCheckbox" id="checkbox1" value="value">
+            <label for="checkbox1">Ja, ich habe die <a href="./data_protection.html">Datenschutzbedingungen</a> gelesen und bin damit einverstanden.</label>
```

Comment:
Please change the link `./data_protection.html` to `./data_protection`. Due to https://github.com/DavidRisch/kicker/pull/4 we do not serve any static html.